cmd/dockerd/docker.go

```diff
@@ -40,6 +56,11 @@ func newDaemonCommand() (*cobra.Command, error) {
 
 	flags := cmd.Flags()
 	flags.BoolP("version", "v", false, "Print version information and quit")
+	if runtime.GOOS == "linux" {
+		flags.BoolVar(&dumpDefaultSeccompProfile, "dump-default-seccomp-profile", false, "Dump the default seccomp profile and quit")
```

Comment:
Perhaps `Print` instead of `Dump` ? Wondering if `dump` may be confusing ("will it drop/unload the profile?)

```suggestion
		flags.BoolVar(&dumpDefaultSeccompProfile, "dump-default-seccomp-profile", false, "Print the default seccomp profile and quit")
```

Reply:
fixed